site/docs/flink.md

```diff
@@ -270,7 +270,7 @@ USE iceberg_db;
 ### `CREATE TABLE`
 
 ```sql
-CREATE TABLE hive_catalog.default.sample (
+CREATE TABLE hive_catalog.`default`.sample (
```

Comment:
Thanks @hililiwei to address this issue,  I think we can quote all the identifiers (include catalog identifier, database name, table name) , that looks more unified.

Reply:
Thanks.Done.